README.md

```diff
@@ -1,32 +1,46 @@
 # flasc_cookiecutter_template
-This repository hosts a Cookiecutter template for wind farm analysis projects using FLASC and FLORIS. It also includes a set of example scripts either with which to learn FLASC, or upon which to base your actual analysis scripts. To open a new project, you will need [Cruft](https://cruft.github.io/cruft/) (recommended) or Cookiecutter.
+This repository hosts a Cookiecutter template for wind farm analysis projects using [FLASC](https://github.com/NREL/flasc/) and [FLORIS](https://github.com/NREL/floris). It also includes a set of example scripts either with which to learn FLASC, or upon which to base your actual analysis scripts. To open a new project, you will need [Cruft](https://cruft.github.io/cruft/) (recommended) or Cookiecutter.
 
 flasc_cookiecutter_template is developed and maintained by NREL, with community and industry support from, among others, Bart Doekemeijer (Shell Global Solutions International B.V.). Initial code architecture by Bart Doekemeijer.
 
 
 ## Creating your own package from this template
-Cookiecutter is the standard and most widely used tool to create packages from templates such as this repository. In contrary to Cruft, Cookiecutter will create a standalone copy of this template that you can modify and add to as you please. Any updates in the upstream Cookiecutter template can not be pulled into your repository, and hence newer functionalities will not be easily available to your package. To keep your package in sync with the template, we suggest you to use Cruft.
+Cookiecutter is the standard and most widely used tool to create packages from templates such as this repository. In contrary to Cruft, Cookiecutter will create a standalone copy of this template that you can modify and add to as you please. Any updates in the upstream Cookiecutter template can not be pulled into your repository, and hence newer functionalities will not be easily available to your package. To keep your package in sync with the template, we suggest you to use Cruft. 
 
-To use Cruft, use
+To create a new conda environment for your project, use
+```
+conda create --name your-project-env python==3.10 pip
+conda activate your-project-env
+```
+
+FLASC is a requirement of flasc_cookiecutter_template. To install FLASC, run
+```
+git clone https://github.com/NREL/flasc
+pip install -e flasc
```

Comment:
One small oddity that I noticed when I first tried to create the project using Cruft in a new conda environment that did not yet contain FLASC: Cruft pip installed fine, but the `cruft create` step failed because numpy was not in the conda environment. It's not immediately clear why numpy would be needed for this step. However, pip installing FLASC before calling `cruft create` prevents this issue, because numpy is installed with FLASC.